src/helio.jl

```diff
@@ -0,0 +1,139 @@
+    m = cirrange(mlong - plong, 2*pi)
+    e1 = m + ((eccen * sin(m) - m)/ (1 - eccen*cos(m)))
+    e = e1 + ((m + eccen * sin(m) - e1)/ (1 - eccen*cos(e1)))
+    maxdiff = abs(e - e1)
+    niter = 0
+
+    while maxdiff> 1e-5 && niter<10
+        e1 = e
+        e = e1 + ((m + eccen * sin(m) - e1)/ (1 - eccen*cos(e1)))
+        maxdiff = abs(e-e1)
+        niter += 1
+    end
+    nu = 2 * atan(sqrt((1 + eccen)/(1 - eccen)) * tan(e/2))
+    hrad = a * (1 - eccen * cos(e))
+    hlong = cirrange(nu + plong, 2 * pi)
+    hlat = asin(sin(hlong - along) * sin(inc))
+    if !radian
+        hlong = rad2deg(hlong)
+        hlat = rad2deg(hlat)
+    end
+    return hrad, hlong, hlat
+end
+
+"""
+    helio(jd, list[, radian=true]) -> hrad, hlong, hlat
+
+### Purpose ###
+
+Compute heliocentric coordinates for the planets.
+
+### Explanation ###
+
+The mean orbital elements for epoch J2000 are used. These are derived
+from a 250 yr least squares fit of the DE 200 planetary ephemeris to a
+Keplerian orbit where each element is allowed to vary linearly with
+time. Useful mainly for dates between 1800 and 2050, this solution fits the
+terrestrial planet orbits to ~25'' or better, but achieves only ~600''
+for Saturn.
+
+### Arguments ###
+
+* `jd`: julian date, scalar or vector
+* `num`: integer denoting planet number, scalar or vector
+  1 = Mercury, 2 = Venus, ... 9 = Pluto
+* `radian`(optional boolean keyword): if this keyword is set to
+  `true`, than the longitude and latitude output are in radian rather than degrees.
+
+### Output ###
+
+* `hrad`: the heliocentric radii, in astronomical units.
+* `hlong`: the heliocentric (ecliptic) longitudes, in degrees.
+* `hlat`: the heliocentric latitudes in degrees.
+
+### Example ###
+
+(1) Find heliocentric position of Venus on August 23, 2000
+
+```jldoctest
+julia> helio(jdcnv(2000,08,23,0), 2)
+(0.7213669177850188, 198.38959174187056, 2.8873974174644363)
+```
+
+(2) Find the current heliocentric positions of all the planets
+
+```jldoctest
+julia> helio.([jdcnv(1900)], [1,2,3,4,5,6,7,8,9])
+9-element Array{Tuple{Float64,Float64,Float64},1}:
+ (0.421663, 203.273, 2.97717)
+ (0.727449, 344.537, -3.39243)
+ (0.983246, 101.55, 0.0126693)
+ (0.360942, 288.126, -1.58013)
+ (5.38753, 235.849, 0.914218)
+ (10.0575, 268.058, 1.08448)
+ (18.9866, 250.014, 0.0535316)
+ (29.8764, 87.07, -1.24511)
+ (47.1407, 74.6294, -9.89928)
+```
+### Notes ###
+
+This program is based on the two-body model and thus neglects
+interactions between the planets.
+
+Additionally, there's a significant difference in the output given
+for the heliocentric radii of Mars, as the mean orbital quantities
+of planet Mars (see [`common`](@ref)) are differnet than what
+is used in IDL AstroLib.
+
+The coordinates are given for equinox 2000 and *not* the equinox
+of the supplied date.
+
+Code of this function is based on IDL Astronomy User's Library.
+"""
+helio(jd::Real, num::Integer, radian::Bool=false) =
```

Comment:
In `moonpos`, `precess`, and `sunpos` the keyword `radians` is used.  This should be probably fixed also in `euler`.